src/dispatch/notification/service.py

```diff
@@ -66,3 +74,31 @@ def delete(*, db_session, notification_id: int):
     )
     db_session.delete(notification)
     db_session.commit()
+
+
+def send(*, db_session, class_instance: Type[Base], notification_params: dict):
+    """Sends notifications."""
+    notifications = get_all(db_session=db_session)
```

Comment:
nit: we should query for all `active/enabled` notifications and remove the check later on.